exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/generic_config_ui.ipynb

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.                        for row in input_group] 
</pre><p>it will be better to unpack every row in for loop, rather than access row's elements by index. The same way you're doing in <code>save_configuration</code> function:</p><p> </p><pre class="ql-syntax" spellcheck="false">[
  widgets.Box([widgets.Label(value=title, ...), widget], layout=ui_look.row_layout)
  for title, widget, _ in input group
  for input_group in inputs
] 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Completely agree, thanks